### PR TITLE
documentation for london-cycle-traffic-air-quality and london-no2

### PR DESCRIPTION
**Pull request by arya-hemanshu**
documentation for london-cycle-traffic-air-quality and london-no2

### CODE REVIEW

File: Samples/london-no2.py [hunk 1]
```diff
@@ -1,19 +1,40 @@
+'''
+This recipe will get you started with using Digital Connector,
```
Comment (lorenaqendro) [inline Samples/london-no2.py]: with the DC, could remove `using`

File: Samples/london-no2.py [hunk 2]
```diff
@@ -1,19 +1,40 @@
+'''
+This recipe will get you started with using Digital Connector, 
+it uses the minimal objects required to built a useful recipe.
```
Comment (lorenaqendro) [inline Samples/london-no2.py]: typo build

File: Samples/london-no2.py [hunk 3]
```diff
@@ -1,19 +1,40 @@
...
+# Note: all three declaration should be relative to user's home directory
 tombolo_path = 'Desktop/TomboloDigitalConnector'
 recipe_output_location = 'Desktop/london-no2.json'
 model_output = 'Desktop/london-no2.geojson'
 
+# importing the required classes for building the recipe
 from recipe import Recipe, Field, Datasource, AttributeMatcher, Subject, Match_Rule, LatestValueField, Dataset
 
+# Creating Subject and Datasources to tell DC which importer needs to be called 
+# inorder to dowload datasets for more info please refer to documentation
```
Comment (lorenaqendro) [inline Samples/london-no2.py]: typo in order
typo download
... to run the recipe. For more info ...

File: Samples/london-no2.py [hunk 4]
```diff
@@ -1,19 +1,40 @@
...
 subjects = Subject(subject_type_label='airQualityControl', provider_label='erg.kcl.ac.uk')
 datasources = Datasource(importer_class='uk.org.tombolo.importer.lac.LAQNImporter', datasource_id='airQualityControl')
 
-attribute_matcher = AttributeMatcher(provider='erg.kcl.ac.uk', label='NO2 40 ug/m3 as an annual me')
+# Creating Attribute to tell DC what exactly does it needs to search in the database
```
Comment (lorenaqendro) [inline Samples/london-no2.py]: typo need

File: Samples/london-cycle-traffic-air-quality.py [hunk 5]
```diff
@@ -1,23 +1,59 @@
+'''
+  This recipe gets three datasources
+     - London Air Quality data by calling the LAQNImporter.java
+     - London Traffic data using TrafficCountImporter.java
+     - London localAuthority data using OaImporter.java
+
+  By using LatestValueField it performs an operations
```
[the remark this answers is not in the document]
Comment (arya-hemanshu) [reply Samples/london-cycle-traffic-air-quality.py]: done

File: Samples/london-cycle-traffic-air-quality.py [hunk 6]
```diff
@@ -1,23 +1,59 @@
...
+# importing the required classes for building the recipe
 from recipe import Recipe, Dataset, Subject, Match_Rule, Datasource, GeographicAggregationField, LatestValueField, AttributeMatcher, ArithmeticField
 
+
+# Creating Subject which has a scope of LocalAuthority and would filter the dataset for London only 
 match_rule = Match_Rule(attribute_to_match_on='label', pattern='E090%')
 main_subject = Subject(subject_type_label='localAuthority', provider_label='uk.gov.ons', match_rule=match_rule)
 
+# Creating Datasources to tell DC which importer needs to be called 
+# inorder to dowload datasets required to run the recipe for more info please refer to documentation
```
[the remark this answers is not in the document]
Comment (arya-hemanshu) [reply Samples/london-cycle-traffic-air-quality.py]: done

File: Samples/london-cycle-traffic-air-quality.py [hunk 7]
```diff
@@ -1,23 +1,59 @@
...
+attribute = AttributeMatcher(provider='erg.kcl.ac.uk', label='NO2 40 ug/m3 as an annual mean')
 l_v_f = LatestValueField(attribute_matcher=attribute)
 parent_field = GeographicAggregationField(subject=pf_subject, label='NitrogenDioxide', function='mean', field=l_v_f)
 
+# Below are creating 3 Fields two LatestValueFields and 1 ArithmeticField
+# ArithmeticField is the parent field which contains two child fields (2 LatestValueFields)
+# First LatestValueField is taking sum of all BicycleCount in a LocalAuthority
+# Second LatestValueField is taking sum of all CarsTaxis in a LocalAuthority
+# Then they both are passed to ArithmeticField which takes the fraction of BicycleCount
+# by dividing first field from second field
```
[the remark this answers is not in the document]
Comment (arya-hemanshu) [reply Samples/london-cycle-traffic-air-quality.py]: done

File: Samples/london-cycle-traffic-air-quality.py [hunk 8]
```diff
@@ -27,9 +63,12 @@
 field_2 = GeographicAggregationField(label='CarCount', subject=a_sub, function='sum', field=sub_field_latest_2)
 arithmetic_field = ArithmeticField(label='BicycleFraction', operation='div', operation_on_field_1=field_1, operation_on_field_2=field_2)
 
+# Passing everything to a Dataset Object as a list and building and running the recipe
```
[the remark this answers is not in the document]
Comment (arya-hemanshu) [reply Samples/london-cycle-traffic-air-quality.py]: done

File: Samples/london-no2.py (hunk 4 above)
Comment (arya-hemanshu) [reply Samples/london-no2.py]: done

File: Samples/london-no2.py (hunk 3 above)
Comment (arya-hemanshu) [reply Samples/london-no2.py]: done

File: Samples/london-no2.py (hunk 2 above)
Comment (arya-hemanshu) [reply Samples/london-no2.py]: done

File: Samples/london-no2.py (hunk 1 above)
Comment (arya-hemanshu) [reply Samples/london-no2.py]: done

**Pull request merged**